RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Pinny Release Checklist
+
+Take this checklist and paste it into your release PR. Ensure steps are followed in order.
+
+- [ ] Tests pass in the terminal and in supported browsers.
+  - See [README](./README.md#browser-compatibility) for list of supported browsers.
```

Comment:
Can you use automation to check the positioning of elements relative to the viewport?